### PR TITLE
docs: add (disabled) diagnostics config to bug report template

### PR DESCRIPTION
**Pull request by alex-courtis**
Drafted during #2460

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/nvt-min.lua [hunk 1]
```diff
@@ -11,6 +11,10 @@ local function load_plugins()
       "wbthomason/packer.nvim",
       "nvim-tree/nvim-tree.lua",
       "nvim-tree/nvim-web-devicons",
+
+      -- UNCOMMENT for diagnostics issues
+      -- "neovim/nvim-lspconfig",
```
Comment (gegoune) [inline .github/ISSUE_TEMPLATE/nvt-min.lua:16]: Shouldn't we request native lsp start function call over additional plugin?

Comment (alex-courtis) [reply .github/ISSUE_TEMPLATE/nvt-min.lua:16]: Yes please!

Do you know how to get that going?

**Review by Akmadan23 — APPROVED** [review]
This makes total sense. Seeing this I even thought to add a separate bug report template for diagnostics related issues but maybe that's too much. This seems an optimal middle ground.

File: .github/ISSUE_TEMPLATE/nvt-min.lua (hunk 1 above)
Comment (gegoune) [reply .github/ISSUE_TEMPLATE/nvt-min.lua:16]: I guess we can just direct to `:h vim.lsp.start` which has an example right there.

Comment (alex-courtis) [reply .github/ISSUE_TEMPLATE/nvt-min.lua]: Well... I was overthinking that; I didn't realise it was that simple.

Please test drive the new commented out template.

**Pull request merged**